### PR TITLE
latest node binaries for windows 

### PR DESCRIPTION
**Pull request by rhtpandeyIN**
latest node binaries for windows and nodejs version < 0.12.x support for node-ibm-db discontinued.

### CODE REVIEW

**Comment by sam-github** [comment]
@bimalkjha Note that dropping support for a node version is a breaking change, so this is semver major. The release this comes out in has to be a major greater than whatever the previous release is.

File: installer/driverInstall.js [hunk 1]
```diff
@@ -36,17 +36,31 @@ var install_node_ibm_db = function(file_url) {
 
             //Windows node binary names should update here.
             var ODBC_BINDINGS = 'build\/Release\/odbc_bindings.node';
-            var ODBC_BINDINGS_V10 = 'build\/Release\/odbc_bindings.node.0.10.36';
             var ODBC_BINDINGS_V12 = 'build\/Release\/odbc_bindings.node.0.12.7';
             var ODBC_BINDINGS_V4 = 'build\/Release\/odbc_bindings.node.4.6.1';
             var ODBC_BINDINGS_V6 = 'build\/Release\/odbc_bindings.node.6.9.1';
 
+            //ERROR: NodeJs version < 0.12.x Support for node-ibm_db Windows has been discontinued.
+            if(Number(process.version.match(/^v(\d+\.\d+)/)[1]) < 0.12){
```
Comment (sam-github) [inline installer/driverInstall.js]: consider using https://www.npmjs.com/package/semver for comparing semantic versions

Comment (rhtpandeyIN) [inline installer/driverInstall.js]: Here we are just checking the locally installed node.Js version in the system, and then comparing it. does it required to use semver package for this?

**Comment by bimalkjha** [comment]
@sam-github This is going to be a 2.0.0 version release. We are removing v0.10.x binary only for Windows. Other platforms should work. Our decision is to provide windows binary only for 4 releases at a time, so now for v 0.12, 4.x, 6.x and 7.x. Thanks.

File: installer/driverInstall.js (hunk 1 above)
Comment (sam-github) [inline installer/driverInstall.js]: There are some reasons having to do with package installation order in npm2 to not use your dependencies in install scripts, though, so perhaps the best thing would be just to implement a `function semverLessThan() ` and use it to do comparisons. It should break the major and minor apart and compare them seperately to not have the "versions miscompared as floats" problem.

**Comment by sam-github** [comment]
@bimalkjha OK. Consider dropping support for 0.12 in 2.x: It is end of lifed by node (no new release, including security releases): https://github.com/nodejs/LTS#lts-schedule

Its also not supported by IBM: https://developer.ibm.com/node/sdk/

Dropping support for it would probably make your life easier.

In this case, I think a properly setup Windows system will still build the addon, its just the pre-built binary you are dropping, right?

**Comment by rhtpandeyIN** [comment]
@sam-github , yes If any one wants to build the module with older versions of nodeJs, they can. We are just removing the pre-built binaries, and discontinue support as well. Also, this building binary part is for windows only, Linux systems can still compile the code. 
We will discuss with our team to discontinue 0.12 also, thanks.

File: installer/driverInstall.js (hunk 1 above)
Comment (rhtpandeyIN) [inline installer/driverInstall.js]: I got your point, thanks. I will look into it.

**Comment by wallali** [comment]
Package.json still says:

```
  "engines": {
    "node": ">=0.10.0"
  }
```

**Comment by bimalkjha** [comment]
On non-windows platform, you can use 0.10.0 too. Only windows has such limitation. Still you can build your own binary on windows if you have visual studio installed. so, no change in package.json. Thanks.

**Pull request merged**